tigerpath/majors_and_certificates/majors/ART_2018.json

```
@@ -0,0 +1,272 @@
+        },
+        {
+          "name": "Practice of Art",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrators in the Practice of Art Track explore the traditions, thought processes, and methods of making visual art in connection with a general program of humanistic education. Courses are offered in painting, drawing, printmaking, sculpture, photography, film and video production, and film history and theory.",
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Two courses in the Program in Visual Arts and one course in the Department of Art and Archaeology. By the first Wednesday following spring break, sophomores submit an application and a portfolio of creative work to the Lewis Center for the Arts administrative office. The admissions committee for the Program in Visual Arts will notify students accepted into the program by early April. No AP credit is accepted toward the Practice of Art concentration.",
+              "req_list": [
+                {
+                  "name": "Visual Arts",
+                  "max_counted": 1,
+                  "min_needed": 2,
+                  "explanation": "Two courses in the Program in Visual Arts.",
+                  "course_list": [
+                    "VIS *"
+                  ]
+                },
+                {
+                  "name": "Art and Archaeology",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "One course in the Department of Art and Archaeology",
+                  "course_list": [
+                    "ART *"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Practice of Art Tracks",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "A total of 10 courses, of which at least seven must be from the Program in Visual Arts and three must be from the Department of Art and Archaeology and taught by Art and Archaeology faculty.",
+              "req_list": [
+                {
+                  "name": "Visual Arts",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": "The visual arts courses must include: studio courses in at least two different media; two studio courses at the 300 or 400 level; VIS 392 Issues in Contemporary Art; and VIS 416 Exhibition Issues and Methods or VIS 417 Fall Film Seminar. The Department of Art and Archaeology courses must include: at least one course from Group 3 (19th century to the present); and two others, including at least one course from Group 1 (ancient) or Group 2 (medieval/early modern). The third course may be from any group.",
+                  "req_list": [
+                    {
+                      "name": "VIS 416/VIS 417",

```

There seems to be an issue with forward slashes not being escaped properly, and this is causing the search for this requirement to fail. 
Eventually we should find and fix the underlying issue, of course, but for now, you can avoid the issue by slightly renaming the requirement.
Something like
```suggestion
                      "name": "VIS 416 or VIS 417",
```